cmd/fleetctl/apply.go

```diff
@@ -27,7 +27,7 @@ type specGroup struct {
 	Teams        []*fleet.TeamSpec
 	Packs        []*fleet.PackSpec
 	Labels       []*fleet.LabelSpec
-	AppConfig    *fleet.AppConfigPayload
+	AppConfig    interface{} // This needs to be interface{} to allow for the patch logic
```

Comment:
This is needed because if we don't do it like this, we'll need to use pointers to be able to reset a certain section and so the patch logic works, because otherwise it's impossible to differentiate between "the user wants this set to false" vs "this is false because it's the zero value for this variable, but the user didn't set this, so leave it as it was before".